app/views/users/change_username.html.erb

```diff
@@ -1,12 +1,23 @@
 <!--Descriptive page name, messages and instructions-->
 <h2 class="heading"><%= ts("Change My User Name") %></h2>
 <%= error_messages_for :user %>
-<p class="caution">
-  <%= ts("Please use this feature with caution. For information on how changing your user name will affect your account, please check out the %{account_faq}. Note that changes to your username may take several days or longer to appear. If you are still seeing your old user name on your works, bookmarks, series, or collections after a week, please %{contact_support}.",
-  account_faq: (link_to ts("Account FAQ"), archive_faqs_path + "/your-account#namechange"),
-  contact_support: (link_to ts("contact Support"), new_feedback_report_path)
-  ).html_safe %>
-</p>
+<div class="caution">
+  <p>
+    <strong><%= ts("Please use this feature with caution.") %></strong>
+    <%= ts("You can change your user name %{times} time every %{days} days.",
+      times: ArchiveConfig.USERNAME_CHANGE["TIMES"],
+      days: ArchiveConfig.USERNAME_CHANGE["INTERVAL_DAYS"]) %>
+    <% if @user.renamed_at %>
+      <%= ts("You last changed your user name on %{renamed_at}.", renamed_at: l(@user.renamed_at, format: :long)) %>
+    <% end %>
+  </p>
+  <p>
+    <%= ts("For information on how changing your user name will affect your account, please check out the %{account_faq}. Note that changes to your username may take several days or longer to appear. If you are still seeing your old user name on your works, bookmarks, series, or collections after a week, please %{contact_support}.",
+    account_faq: (link_to ts("Account FAQ"), archive_faqs_path + "/your-account#namechange"),
```

Comment:
Because the variable represents a hyperlink, can you rename it [with the suffix _link](https://github.com/otwcode/otwarchive/wiki/Internationalization-(i18n)-Standards)?